DataFormats/VertexSoA/interface/ZVertexDevice.h

```diff
@@ -9,18 +9,18 @@
 #include "DataFormats/VertexSoA/interface/ZVertexHost.h"
 #include "DataFormats/Portable/interface/PortableDeviceCollection.h"
 
-template <int32_t S, typename TDev>
-class ZVertexDeviceSoA : public PortableDeviceCollection<reco::ZVertexLayout<>, TDev> {
+template <int32_t NVTX, int32_t NTRX, typename TDev>
```

Comment:
what does "X" in NTRX stand for? isn't it a shortened word "track"?

Reply:
Yes. Better expand it?

Reply:
Thanks for the review @slava77 I chased all instances of `trx` in the commits and replaced with `trk` (also in the original PR).